docker/worker/entrypoint.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+set -e
+
+zypper --non-interactive in qemu-ovmf-x86_64 || zypper --non-interactive in qemu-uefi-aarch64 || exit 1
```

Comment:
I'd keep this in `Dockerfile` instead.

Reply:
Thinking about that:

The problem is that if we move this to the dockerfile the package is going to be used depends on the architecture of the machine we are doing the build. So..

2 options:
- We could create images for each architecture. To do that we need to build every image using the specific architecture. During the OBS for instance if is not done in this way already.
- Install them during the execution process

Reply:
Option number three would be to determine the architecture before installing the package in `Dockerfile`. Something like: 
```bash
[[ "$(uname -m)" =~ "aarch64" ]] && zypper --non-interactive in qemu-uefi-aarch64
```

Reply:
That would definitely be better. I really don't like the "let's try some x86_64-specific stuff and if that doesn't work *for any reason* fall back to the *only* other architecture aarch64".

Reply:
Ok, I am removing this commit and left all this stuff in the Dockerfile but then we'll need to build in every architecture and maybe tag this as openqa-webui:x86 ....

Reply:
And the build could be 
`docker build -t openqa-worker:$(uname -m) .`

Reply:
> And the build could be
> `docker build -t openqa-worker:$(uname -m) .`

I don't think so. You can have multiple containers of different architectures with the same tag.